Formula/g/gt.rb

```diff
@@ -0,0 +1,25 @@
+class Gt < Formula
```

Comment:
It seems it's called `gittool` everywhere, so the formula should use the same name.